app/org/sagebionetworks/bridge/dynamodb/DynamoInitializer.java

```diff
@@ -73,12 +93,55 @@ static void deleteTable(Class<?> table) {
                 // Must be active to be deleted
                 DynamoUtils.waitForActive(DYNAMO, tableDscr.getTableName());
             }
-            logger.info("Deleting table " + tableName);
+            LOG.info("Deleting table " + tableName);
             DYNAMO.deleteTable(tableName);
             DynamoUtils.waitForDelete(DYNAMO, tableDscr.getTableName());
-            logger.info("Table " + tableName + " deleted.");
+            LOG.info("Table " + tableName + " deleted.");
         } catch (ResourceNotFoundException e) {
-            logger.warn("Table " + tableName + " does not exist.");
+            LOG.warn("Table " + tableName + " does not exist.");
+        }
+    }
+
+    static void backupPipelineForTables(final Collection<TableDescription> tables) {
+        Environment env = CONFIG.getEnvironment();
+
+        String envAndUser = env.name().toLowerCase() + "-" + CONFIG.getUser();
+
+        String pipelineName = "Dynamo backup for bridge: " + envAndUser;
+        String pipelineUniqueId = "dynamo-backup-bridge." + envAndUser;
+        String s3Bucket = DYNAMO_BACKUP_BUCKET_PREFIX + envAndUser;
+
+        List<PipelineObject> pipelineObjects =
+                DynamoDataPipelineHelper.createPipelineObjects(DYNAMO_REGION, tables, s3Bucket, BACKUP_SCHEDULED_TIME);
+
+        LOG.debug("Pipeline objects: " + pipelineObjects.toString());
+
+        CreatePipelineResult createPipelineResult = DATA_PIPELINE_CLIENT.createPipeline(new CreatePipelineRequest()
+                .withName(pipelineName)
+                .withUniqueId(pipelineUniqueId));
+
+        String pipelineId = createPipelineResult.getPipelineId();
+
+        LOG.info("Updating backup pipeline, pipelineId=" + pipelineId);
+
+        PutPipelineDefinitionRequest putPipelineDefinitionRequest =
+                new PutPipelineDefinitionRequest().withPipelineId(pipelineId).withPipelineObjects(pipelineObjects);
```

Comment:
What happens if the pipeline already exists and has backup jobs in it? Can you add a comment describing what happens?